health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/AppConfigurationParentLayer.js

```
@@ -0,0 +1,384 @@
+    if (showToast) {
+      setTimeout(closeToast, 10000);
+    }
+  }, [showToast]);
+  useEffect(() => {
+    if (formData || formId) {
+      parentDispatch({
+        key: "SETFORM",
+        data: convertDataFormat(formData),
+      });
+    } else if (!isLoadingAppConfigMdmsData && AppConfigMdmsData?.[masterName]) {
+      setAppTemplate([...AppConfigMdmsData?.[masterName]]);
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)